support/scripts/mklinux.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+# Linux arm64 boot header
+# https://www.kernel.org/doc/Documentation/arm64/booting.txt
+LINUX_ARM64_HDR = {
+    "CODE0"                   : [0x0000000091005a4d, 0x04],  # Offset   0
+    "CODE1"                   : [0x0000000000000000, 0x04],  # Offset   4
+    "LOAD_OFFS"               : [0x0000000000000000, 0x08],  # Offset   8
+    "IMAGE_SIZE"              : [0x0000000000000000, 0x08],  # Offset  16
+    "KERNEL_FLAGS"            : [0x0000000000000000, 0x08],  # Offset  24
+    "RES2"                    : [0x0000000000000000, 0x08],  # Offset  32
+    "RES3"                    : [0x0000000000000000, 0x08],  # Offset  40
+    "RES4"                    : [0x0000000000000000, 0x08],  # Offset  48
+    "MAGIC"                   : [0x00000000644d5241, 0x04],  # Offset  56
```

Comment:
Unless you are going to change this in the future at runtime, which I doubt, perhaps it would be better to surround it with `()` isntead.

Reply:
Ok